GVRf/Framework/gradle.properties

```diff
@@ -15,6 +15,7 @@
 #Thu May 05 12:55:40 PDT 2016
 android.useDeprecatedNdk=true
 org.gradle.jvmargs=-Xmx2048M
+GVRF_BACKEND=backend_oculus
```

Comment:
I am tempted to say they should but for now I will make the change you requested.